CI is failing for experimental tests: AttributeError: 'GeometricMixtureWrapper' object has no attribute 'get_experts_implementation'

CI is failing for experimental tests: https://github.com/huggingface/trl/actions/runs/29470350425/job/87532140222
> AttributeError: 'GeometricMixtureWrapper' object has no attribute 'get_experts_implementation'

```python
  FAILED tests/experimental/test_nash_md_trainer.py::TestNashMDTrainer::test_nash_md_trainer_training[standard_prompt_only] - AttributeError: 'GeometricMixtureWrapper' object has no attribute 'get_experts_implementation'
  FAILED tests/experimental/test_nash_md_trainer.py::TestNashMDTrainer::test_nash_md_trainer_training[conversational_prompt_only] - AttributeError: 'GeometricMixtureWrapper' object has no attribute 'get_experts_implementation'
  FAILED tests/experimental/test_nash_md_trainer.py::TestNashMDTrainer::test_train_with_peft - AttributeError: 'GeometricMixtureWrapper' object has no attribute 'get_experts_implementation'
  FAILED tests/experimental/test_nash_md_trainer.py::TestNashMDTrainer::test_train_with_peft_and_ref_model - AttributeError: 'GeometricMixtureWrapper' object has no attribute 'get_experts_implementation'
  FAILED tests/experimental/test_nash_md_trainer.py::TestNashMDTrainer::test_train_pre_pefted_model_implicit_ref_with_reward_model - AttributeError: 'GeometricMixtureWrapper' object has no attribute 'get_experts_implementation'
```

Traceback:
```python
   >       trainer.train()
  
  tests/experimental/test_nash_md_trainer.py:193: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  .venv/lib/python3.13/site-packages/transformers/trainer.py:1437: in train
      return inner_training_loop(
  .venv/lib/python3.13/site-packages/transformers/trainer.py:1519: in _inner_training_loop
      self._run_epoch(
  .venv/lib/python3.13/site-packages/transformers/trainer.py:1747: in _run_epoch
      tr_loss_step = self.training_step(model, inputs, num_items_in_batch)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  trl/experimental/nash_md/nash_md_trainer.py:463: in training_step
      model_output, mixture_output = self._generate_completions(model, prompts)
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  trl/experimental/nash_md/nash_md_trainer.py:283: in _generate_completions
      mixture_output = mixture_model.generate(
  .venv/lib/python3.13/site-packages/torch/utils/_contextlib.py:124: in decorate_context
      return func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  .venv/lib/python3.13/site-packages/transformers/generation/utils.py:2681: in generate
      result = decoding_method(
  .venv/lib/python3.13/site-packages/transformers/generation/utils.py:2888: in _sample
      with self._optimize_model_for_decode():
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  /__w/_tool/Python/3.13.14/x64/lib/python3.13/contextlib.py:141: in __enter__
      return next(self.gen)
             ^^^^^^^^^^^^^^
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  self = <trl.experimental.nash_md.nash_md_trainer.GeometricMixtureWrapper object at 0x7facd8fe6ea0>
  
      @contextmanager
      def _optimize_model_for_decode(self: "GenerativePreTrainedModel"):
          # On non-CPU devices, 'batched_mm' can trade off a bit of memory (by duplicating selected experts weights)
          # for much better speed during decoding, especially for smaller inputs. On CPU, grouped_mm is usually better.
          # The MoE may live in a submodel (e.g. a VLM's `text_config`), not the top-level config, so we look at the
          # implementation of every (sub)config and let `set_experts_implementation` recurse the switch back and forth.
  >       original_experts_implementation = self.get_experts_implementation()
                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  E       AttributeError: 'GeometricMixtureWrapper' object has no attribute 'get_experts_implementation'
  
  .venv/lib/python3.13/site-packages/transformers/generation/utils.py:2197: AttributeError
```

--- a/trl/experimental/nash_md/nash_md_trainer.py
+++ b/trl/experimental/nash_md/nash_md_trainer.py
@@ -102,6 +102,12 @@ class GeometricMixtureWrapper(GenerationMixin):
     def _validate_model_kwargs(self, model_kwargs):
         return self.model._validate_model_kwargs(model_kwargs)
 
+    def get_experts_implementation(self):
+        return self.model.get_experts_implementation()
+
+    def set_experts_implementation(self, experts_implementation):
+        self.model.set_experts_implementation(experts_implementation)
+
 
 class NashMDTrainer(OnlineDPOTrainer):
     """

